roles/gitserver/defaults/main.yml

```diff
@@ -1,3 +1,12 @@
 # array of local keys to add
 local_ssh_keys: []
 git_user_password: WFltGBxB/v4Fs
+
+gitserver_repos: []
```

Comment:
@vmcj do we want our DOMjudge repos here as well?

Reply:
Yes